hredis/hredis_test.go

```diff
@@ -30,6 +33,11 @@ func TestRedissURL(t *testing.T) {
 			if err != nil && !cs.shouldErr {
 				t.Fatalf("wanted nil error but got: %v", err)
 			}
+
+			_, err = url.Parse(u)
+			if err != nil {
+				t.Fatal(err)
+			}
```

Comment:
Given each test case what do you expect scheme to be?

Reply:
Fixed in https://github.com/heroku/x/pull/32/commits/e7c2058bd7518a2b4fbef0f90815bc87524b9b05